### DISCUSSION TITLE
Making all properties queryable

### DISCUSSION THREAD

**Discussion by zvfcs** [ldproxy/ldproxy#1254] [Configuration] [answered]
According to the [Feature Collections - Queryables documentation](https://docs.ldproxy.net/services/building-blocks/feature_collections_-_queryables.html), it should be possible to use 
```
include: 
  - *
```

to make all properties queryable. However, I get an error message when doing so. My config is the same as the example in the documentation, so I'm not sure what I'm doing wrong here? Using an actual property name works fine.

![image](https://github.com/user-attachments/assets/b8151cf4-b82d-47fc-87bd-a157185d7004)

**Comment by cportele** [comment, marked as answer]
`*` is a special character in YAML that needs to be quoted (i.e., `- "*"`). 

(I will update the example.)
